packages/mux-video/src/index.ts

```diff
@@ -236,7 +236,11 @@ class MuxVideoElement
   }
 
   get metadata() {
-    return this.__metadata;
+    return Object.assign(this.__metadata, {
+      video_id: this.getAttribute(Attributes.METADATA_VIDEO_ID),
```

Comment:
blocking: This feels a bit out of scope, but also the refactor feels brittle. I think for each of these we need to minimally check `hasAttribute` so that we don't unintentionally overwrite these key/value pairs in `metadata`

Reply:
good catch, we should do some checks here and determine which has precedence.
or maybe update `this.__metadata[key]` in the attributeChangedCallback, that keeps the order I think.